protocols/imap/src/main/java/org/apache/james/imap/processor/fetch/EnvelopeBuilder.java

```diff
@@ -65,33 +65,28 @@ private String headerValue(Headers message, String headerName) throws MailboxExc
         final Header header = MessageResultUtils.getMatching(headerName, message.headers());
         if (header == null) {
             return null;
-        } else {
-            final String value = header.getValue();
-            if (value == null || "".equals(value)) {
-                return null;
-            } else {
-
-                // ENVELOPE header values must be unfolded
-                // See IMAP-269
-                //
-                //
-                // IMAP-Servers are advised to also replace tabs with single spaces while doing the unfolding. This is what javamails
-                // unfold does. mime4j's unfold does strictly follow the rfc and so preserve them
-                //
-                // See IMAP-327 and https://mailman2.u.washington.edu/mailman/htdig/imap-protocol/2010-July/001271.html
-                return MimeUtility.unfold(value);
-
-            }
         }
+        final String value = header.getValue();
+        if (value == null || "".equals(value)) {
```

Comment:
not related to your change but line 110 we compare the trimmed value. Here not, shoud it be trimmed to ?

Reply:
(no idea)